tasks/main.yml

```diff
@@ -3,6 +3,10 @@
   include_vars: "{{ ansible_distribution }}.yml"
   when: ansible_distribution == 'FreeBSD' or ansible_distribution == 'Fedora'
 
+- name: Include OS-specific variables for Amazon.
+  include_vars: "{{ ansible_distribution }}-{{ ansible_distribution_version}}.yml"
```

Comment:
It seems like we should either have this task, and the associated vars files, or the other task ('version-specific') and associated vars files, but not both. They seem to duplicate the effort.

Reply:
Yea, I can see what I've done there, have updated to for the former